conanfile.py

```diff
@@ -12,7 +12,9 @@ class LibMP3LameConan(ConanFile):
     url = "https://github.com/bincrafters/conan-libname"
     description = "LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL."
     homepage = "http://lame.sourceforge.net/"
-    license = "LGPL"
+    author = "Bincrafters <bincrafters@gmail.com>"
+    topics = "multimedia", "audio", "libmp3lame", "mp3", "decoder", "encoding", "decoding"
```

Comment:
please use braces, and the first topic should always be `conan` followed by the lib name

`topics = ("conan", "libname", "multimedia".....)`